experimental/uefi/channel/src/lib.rs

```diff
@@ -74,6 +74,7 @@ where
         };
         self.inner.write_all(&length_bytes)?;
         self.inner.write_all(&frame.body)?;
+        self.inner.flush()?;
```

Comment:
Should we add #2876 here?

Reply:
This was added so that we don't need to update this code when implementing #2876. We would still flush for every frame, but not for every invocation of `write_all`.